bandit-github/action.yml

```
@@ -0,0 +1,45 @@
+name: 'Bandit Github Action'
+description: 'Bandit Github Action'
+inputs:
+  path:
+    description: 'File or directory to run bandit on'
+    required: false
+    default: '.'
+  level:
+    description: 'Report only issues of a given severity level or higher. Can be LOW, MEDIUM or HIGH. Default is UNDEFINED (everything)'
+    required: false
+    default: 'UNDEFINED'
+  confidence:
+    description: 'Report only issues of a given confidence level or higher. Can be LOW, MEDIUM or HIGH. Default is UNDEFINED (everything)'
+    required: false
+    default: 'UNDEFINED'
+  excluded_paths:
+    description: 'comma-separated list of paths (glob patterns supported) to exclude from scan (note that these are in addition to the excluded paths provided in the config file) (default: .svn,CVS,.bzr,.hg,.git,__pycache__,.tox,.eggs,*.egg)'
+    required: false
+    default: 'DEFAULT'
+  exit_zero:
+    description: 'exit with 0, even with results found'
+    required: false
+    default: 'DEFAULT'
+  skips:
+    description: 'comma-separated list of test IDs to skip'
+    required: false
+    default: 'DEFAULT'
+  ini_path:
+    description: 'path to a .bandit file that supplies command line arguments'
+    required: false
+    default: 'DEFAULT'
+  GITHUB_TOKEN:
+    description: 'Github token of the repository (automatically created by Github)'
+    required: true
+runs:
+  using: 'docker'
+  image: 'Dockerfile'
```

This image takes a while to build. Use a prebuild Image instead:
```suggestion
runs:
  using: 'docker'
  image: 'docker://ghcr.io/snapaddy/actions-bandit-github:1.2.0'
```

Also requires changes in:
- docker-bake.hcl
- .github/workflows/release.yaml